DOCS-3060: Update documentation on discounts (#2015)
MultiSafepay/docs: content/api/orders/discount-pay-later-orders.md

---
weight: 215
meta_title: "API reference - Discount pay later orders - MultiSafepay Docs"
---
{{< description >}}
### Discount pay later orders

> To add a discount to your [pay later](/payments/methods/pay-later/) order:

Reviewer: ```suggestion
To discount a [pay later](/payments/methods/pay-later/) order, either:
```

> - Reduce the `amount` of the order.

Reviewer: ```suggestion
- Reduce the total order `amount`, **OR**
```

> - Reduce the `unit_price` of items in the shopping cart that are discounted.

Reviewer: ```suggestion
- Reduce the `unit_price` of the items in the shopping cart you want to discount.
```

- Decimal currencies: Value for 10 EUR = 1000 (1000 cents)
- Zero-decimal currencies: Value for ¥10 = 10 

> **Note**: This amount includes the discount.

Reviewer: ```suggestion
**Note:** This amount includes the discount.
```
